### ISSUE TITLE
pl-order-blocks displaying incorrect "correct answer" in answer panel

### ISSUE THREAD

**Issue by bowmnath** [PrairieLearn/PrairieLearn#5665]
The "correct answer" displayed by `pl-order-blocks` is sometimes not a valid correct answer.  I have not yet noticed a particular pattern in how it fails, but here is a very simple MWE `question.html`:

```html
<pl-order-blocks answers-name="order" grading-method="ranking" partial-credit="none">
  <pl-answer ranking=5>1</pl-answer>
  <pl-answer ranking=4>2</pl-answer>
  <pl-answer ranking=3>3</pl-answer>
  <pl-answer ranking=2>4</pl-answer>
  <pl-answer ranking=1>5</pl-answer>
</pl-order-blocks>
```

When I tested that question on the live server, it displayed the correct answer as `1, 2, 3, 4, 5`, when the only valid answer is actually `5, 4, 3, 2, 1`.  

Note that the element is still **grading** correctly.  In this case, it (correctly) accepted `5, 4, 3, 2, 1` and (correctly) did not accept `1, 2, 3, 4, 5`.

**Comment by SethPoulsen** [comment]
Ya, this is a known issue that has been on my list of things to fix for a long time, I just haven't gotten to it. Actually, for a long time it wasn't possible to do because we couldn't guarantee a solution to problems in the `dag` grading mode, but now we can so it is totally fixable at this point.

`pl-order-blocks` just assumes that the order you write the items in the question HTML is a correct solution when it goes to display the correct solution.

I don't think there is a dedicated issue for this so we'll keep it open, but I do mention it over in #4650

**Comment by SethPoulsen** [comment]
@bowmnath I've opened a PR to fix this at #5871. Feel free to leave it a code review if you want to help speed the process up!
